src/stk/molecular/key_makers/smiles.py

```diff
@@ -0,0 +1,44 @@
+"""
+SMILES
+======
+
+"""
+
+from .molecule import MoleculeKeyMaker
+from .utilities import get_smiles
+
+
+class Smiles(MoleculeKeyMaker):
+    """
+    Used to get the SMILES of molecules.
+
+    Examples
+    --------
+    You want to use the isomeric, canonical SMILES from RDKit as part
+    of a JSON representation of a molecule
```

Comment:
Each example should have an italicised title of no more than 72 characters, including the newline at the end. For example, `*Adding SMILES to a Molecule's JSON Representation*` - the stars on either side make the title render as italic in the compiled docs

Reply:
(I realized, just now, that the other key makers do not have titles for their examples - this is an oversight on my part, look at the examples of topology graphs like Linear Polymer to see what I want)